watertap/property_models/seawater_ion_generic.py

```diff
@@ -0,0 +1,262 @@
+            ('Mg_2+, Cl_-', 'Ca_2+, Cl_-'): 0.322,
+            ('Na_+, SO4_2-', 'Ca_2+, SO4_2-'): -0.761,
+            ('Ca_2+, SO4_2-', 'Na_+, SO4_2-'): 0.368,
+            ('Na_+, SO4_2-', 'Mg_2+, SO4_2-'): -0.327,
+            ('Mg_2+, SO4_2-', 'Na_+, SO4_2-'): 0.799,
+            ('Ca_2+, SO4_2-', 'Mg_2+, SO4_2-'): 0,
+            ('Mg_2+, SO4_2-', 'Ca_2+, SO4_2-'): 0.383,
+            ('Na_+, Cl_-', 'Na_+, SO4_2-'): 0.803,
+            ('Na_+, SO4_2-', 'Na_+, Cl_-'): -0.634,
+            ('Ca_2+, Cl_-', 'Ca_2+, SO4_2-'): 0,
+            ('Ca_2+, SO4_2-', 'Ca_2+, Cl_-'): -0.264,
+            ('Mg_2+, Cl_-', 'Mg_2+, SO4_2-'): -0.707,
+            ('Mg_2+, SO4_2-', 'Mg_2+, Cl_-'): -0.841
+            }},
+    "default_scaling_factors": {("flow_mol_phase_comp", ("Liq", "Na_+")): 1e1,
+                                ("flow_mol_phase_comp", ("Liq", "Ca_2+")): 1e3,
+                                ("flow_mol_phase_comp", ("Liq", "Mg_2+")): 1e2,
+                                ("flow_mol_phase_comp", ("Liq", "SO4_2-")): 1e2,
+                                ("flow_mol_phase_comp", ("Liq", "Cl_-")): 1e1,
+                                ("flow_mol_phase_comp", ("Liq", "H2O")): 1e-1,
+                                ("mole_frac_comp", "Na_+"): 1e2,
+                                ("mole_frac_comp", "Ca_2+"): 1e4,
+                                ("mole_frac_comp", "Mg_2+"): 1e3,
+                                ("mole_frac_comp", "SO4_2-"): 1e3,
+                                ("mole_frac_comp", "Cl_-"): 1e2,
+                                ("mole_frac_comp", "H2O"): 1,
+                                ("mole_frac_phase_comp", ("Liq", "Na_+")): 1e2,
+                                ("mole_frac_phase_comp", ("Liq", "Ca_2+")): 1e4,
+                                ("mole_frac_phase_comp", ("Liq", "Mg_2+")): 1e3,
+                                ("mole_frac_phase_comp", ("Liq", "SO4_2-")): 1e3,
+                                ("mole_frac_phase_comp", ("Liq", "Cl_-")): 1e2,
+                                ("mole_frac_phase_comp", ("Liq", "H2O")): 1,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "NaCl")): 1e1,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "Na2SO4")): 1e2,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "CaCl2")): 1e2,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "CaSO4")): 1e3,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "MgCl2")): 1e2,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "MgSO4")): 1e3,
+                                ("flow_mol_phase_comp_apparent", ("Liq", "H2O")): 1e-1,
+                                ("mole_frac_phase_comp_apparent", ("Liq", "NaCl")): 1e3,  # TODO: these seem to be 1 orders of magnitude too low
+                                ("mole_frac_phase_comp_apparent", ("Liq", "Na2SO4")): 1e4,
```

Comment:
You left a TODO note here. Does this need correction or should the TODO just be removed?

Reply:
This TODO was from someone else so I left it in, in case that person wanted to revisit.